### PR TITLE
develop -> main

### PR DESCRIPTION
**Pull request by Amuhar**

### CODE REVIEW

File: src/keys-api/interfaces/GroupedByModuleOperatorListResponse.ts
Comment (infloop) [inline src/keys-api/interfaces/GroupedByModuleOperatorListResponse.ts:1]: Pascal case files are not following naming convention here

File: src/keys-api/interfaces/KeyListResponse.ts
Comment (infloop) [inline src/keys-api/interfaces/KeyListResponse.ts:1]: Pascal case file name

File: src/keys-api/keys-api.service.ts [hunk 1]
```diff
@@ -39,19 +41,31 @@ export class KeysApiService {
...
-  public async getModulesList() {
-    const result = await this.fetch<SRModuleListResponse>('/v1/modules');
-    if (!result.data?.length || !result.elBlockSnapshot)
-      throw Error('Keys API not synced, please wait');
+  /**
+   *
+   * @param The /v1/keys/find API endpoint returns keys along with their duplicates
+   * @returns
+   */
+  public async findKeysEntires(pubkeys: string[]) {
```
Comment (infloop) [inline src/keys-api/keys-api.service.ts:49]: Looks like there is a convention for naming class methods that fetches data from KAPI  - they start with `get` and then follows with `what we are getting`. This method has a typo and starts with `find` and has strange `entries` in the name. I think it's better to rename it to `getKeysByPubkeys`.

File: src/keys-api/interfaces/GroupedByModuleOperatorListResponse.ts
Comment (Amuhar) [reply src/keys-api/interfaces/GroupedByModuleOperatorListResponse.ts:1]: all interfaces named like this in council, that is why

File: src/keys-api/interfaces/KeyListResponse.ts
Comment (Amuhar) [reply src/keys-api/interfaces/KeyListResponse.ts:1]: the same

File: src/keys-api/keys-api.service.ts (hunk 1 above)
Comment (Amuhar) [reply src/keys-api/keys-api.service.ts:49]: okay, will fix

**Review by infloop — DISMISSED** [review]

**Review by infloop — APPROVED** [review]

**Review by choooze — APPROVED** [review]

**Pull request merged**